Src/HALAL/Services/Communication/FDCAN/FDCAN.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+ *  FDCAN.hpp
+ *
+ *  Created on: 5 nov. 2022
+ *      Author: Pablo
+ */
+#include "Communication/FDCAN/FDCAN.hpp"
+
+#ifdef HAL_FDCAN_MODULE_ENABLED
+
+unordered_map<uint8_t, FDCAN::Instance*> FDCAN::registered_fdcan = {};
+
+unordered_map<FDCAN::DLC, uint8_t> FDCAN::dlc_to_len = {{DLC::bytes_0, 0}, {DLC::bytes_1, 1}, {DLC::bytes_2, 2}, {DLC::bytes_3, 3}, {DLC::bytes_4, 4},
+														{DLC::bytes_5, 5}, {DLC::bytes_6, 6}, {DLC::bytes_7, 7}, {DLC::bytes_8, 8}, {DLC::bytes_12, 12},
+														{DLC::bytes_16, 16}, {DLC::bytes_20, 20}, {DLC::bytes_24, 24}, {DLC::bytes_32, 32}, {DLC::bytes_48, 48},
+														{DLC::bytes_64, 64}
+													    };
+
+optional<uint8_t> FDCAN::inscribe(FDCAN::Peripheral& fdcan){
+	if (!FDCAN::available_fdcans.contains(fdcan)) {
+		ErrorHandler("El fdcan %d no es un fdcan válido.", fdcan);
```

Comment:
Los comentarios en ingles todos, cambialos.
```suggestion
		ErrorHandler("FDCAN %d is not a valid instance.", fdcan);
```